spec/features/support/providers/viewing_a_user_spec.rb

```diff
@@ -13,15 +13,16 @@
   end
 
   scenario "i am directed to the user show page" do
-    then_i_am_directed_to_the_user_show_page
+    then_i_am_directed_to_the_provider_user_show_page
   end
 
   def when_i_visit_the_support_provider_show_page
     support_provider_show_page.load(id: @user.providers.first.id)
   end
 
   def and_there_is_a_user
-    @user = create(:user, :with_provider)
+    @user = create(:user, :with_provider, last_login_date_utc: Time.zone.yesterday)
```

Comment:
This is hiding a problem, as not everyone will have a `last_login_date_utc`value

Reply:
yup agreed, have split out the scenarios